raft/raft.go

```diff
@@ -1607,16 +1606,21 @@ func (r *raft) abortLeaderTransfer() {
 // If the new entries would exceed the limit, the method returns false. If not,
 // the increase in uncommitted entry size is recorded and the method returns
 // true.
+// Configuration changes are never refused.
```

Comment:
Instead of concerning ourselves with the size of the payload and trying to enforce this by allowing zero-byte payloads, could we just check the entry's type directly? We may need to change this condition to "Configuration changes that are alone are never refused".

Reply:
Or is it the case that we also want to allow the empty entry that new leaders propose and this seems like the cleanest way to do it? If that's the case then "Configuration changes are never refused" isn't accurate, right? What we actually want to say is that "entries without a payload are never refused".

Reply:
Yes, in both cases it's the empty payload. I clarified the comment.